extension/app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -52,6 +52,13 @@ export class SimulationModeEthereumClientService {
 		return newSimulation
 	}
 
+	public getSimulationResults = () => {
+		if (this.simulationState === undefined) return []
+		return this.simulationState.simulatedTransactions.map( (x) => {
+			return { ...x.unsignedTransaction, ...x.multicallResponse, realizedGasPrice: x.realizedGasPrice }
+		})
```

Comment:
Style: For lambdas that just return an object and do nothing else, consider dropping the return statement to simplify it a bit.  Note: When the expression is just an object, you must surround it with `()` to ensure the `{}` aren't interpreted as a code block.
```suggestion
		return this.simulationState.simulatedTransactions.map((x) => ({ ...x.unsignedTransaction, ...x.multicallResponse, realizedGasPrice: x.realizedGasPrice }))
```